src/raiden_libs/matrix.py

```diff
@@ -28,10 +29,12 @@
 )
 from raiden.network.transport.utils import timeout_exponential_backoff
 from raiden.settings import (
+    DEFAULT_MATRIX_KNOWN_SERVERS,
     DEFAULT_TRANSPORT_MATRIX_RETRY_INTERVAL,
     DEFAULT_TRANSPORT_RETRIES_BEFORE_BACKOFF,
 )
 from raiden.storage.serialization.serializer import MessageSerializer
+from raiden.utils.cli import get_matrix_servers
```

Comment:
I'm a bit surprised this test passes https://github.com/raiden-network/raiden-services/pull/576/files#diff-043a8444ad20f9b349ab2ba851e6a229L146

But whatever, if it passes it doesn't seem to be necessary.